src/main/java/com/microsoft/cse/helium/HeliumRouter.java

```diff
@@ -0,0 +1,21 @@
+package com.microsoft.cse.helium;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
```

Comment:
I had deleted this file since we were going with the @RestController class. Any particular reason you are bringing this back?

Reply:
No, it was deleted post my branch.  I should have showed a conflict and allowed me to exclude it.  Not sure how to remove it from the PR.  I have deleted it in the latest push.